NEWS.md

```diff
@@ -704,6 +704,9 @@ Deprecated or removed
     have been deprecated in favor of `spdiagm(d => x)` and `spdiagm(d[1] => x[1], d[2] => x[2], ...)`
     respectively. The new `spdiagm` implementation now always returns a square matrix ([#23757]).
 
+  * `spones(A::AbstractSparseArray{T})` has been deprecated in favor of
+    `LinAlg.fillstored!(copy(A), one(T))` ([#TBD]).
```

Comment:
The `one(T)` could simplify to `1` in the vast majority of cases, as `fillstored!` necessarily converts that `1` to `eltype(A)`? Suggesting the simplest replacements first is nice :).